packages/storybook-utils/src/theme.ts

```diff
@@ -90,50 +91,22 @@ const defineTheme = (colors: {
 };
 
 /** All available Storybook breakpoints / viewports supported by onyx. */
-export const ONYX_BREAKPOINTS = {
-  "2xs": {
-    name: "2xs",
-    styles: {
-      width: "320px",
-      height: "100%",
-    },
+export const ONYX_BREAKPOINTS = Object.entries(RAW_ONYX_BREAKPOINTS).reduce(
+  (obj, [name, width]) => {
+    const breakpoint = name as OnyxBreakpoint;
+    obj[breakpoint] = { name: breakpoint, styles: { width: `${width}px`, height: "100%" } };
+    return obj;
   },
-  xs: {
-    name: "xs",
-    styles: {
-      width: "576px",
-      height: "100%",
-    },
-  },
-  sm: {
-    name: "sm",
-    styles: {
-      width: "768px",
-      height: "100%",
-    },
-  },
-  md: {
-    name: "md",
-    styles: {
-      width: "992px",
-      height: "100%",
-    },
-  },
-  lg: {
-    name: "lg",
-    styles: {
-      width: "1440px",
-      height: "100%",
-    },
-  },
-  xl: {
-    name: "xl",
-    styles: {
-      width: "1920px",
-      height: "100%",
-    },
-  },
-} as const;
+  {} as Record<OnyxBreakpoint, StorybookBreakpoint>,
+);
```

Comment:
I prefer to avoid `.reduce` where possible as it requires a higher cognitive overhead.
In this case we can also easily reuse the entries syntax.
```suggestion
export const ONYX_BREAKPOINTS = Object.fromEntries(
  Object.entries(RAW_ONYX_BREAKPOINTS).map(([breakpoint, width]) => [
    breakpoint,
    { name: breakpoint, styles: { width: `${width}px`, height: "100%" } },
  ]),
) as Record<OnyxBreakpoint, StorybookBreakpoint>;
```

Reply:
To be honest I think your suggested syntax requires even more cognitive complexity^^
I'd say `.reduce()` is way more common and known to devs than `Object.fromEntries()`

Reply:
And the `.reduce()` is more type safe since it ensures that the value of each breakpoints actually is of type `StorybookBreakpoint` while `as Record<OnyxBreakpoint, StorybookBreakpoint>;` just forces TypeScript to use this type for `ONYX_BREAKPOINTS `

Reply:
I thought so, too. But now (https://betterprogramming.pub/think-again-before-you-use-array-reduce-28f785b5aea9) I pretty convinced that `fromEntries` is preferably syntax-wise. But you are right, that the type-support of `fromEntries` needs to be improved